### PR TITLE
feat: add simple validation for the SignUP

### PR DESCRIPTION
**Pull request by AdrianValdes**

### CODE REVIEW

File: src/helpers/validateSignUp.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const validatePasswordsEquality = (password, repeatedPassword) =>
+  password === repeatedPassword;
+
+const checkPassword = (password) => {
+  // at least one number, one lowercase and one uppercase letter
+  // at least six characters
+  const re = /(?=.*\d)(?=.*[a-z])(?=.*[A-Z]).{6,}/;
```
Comment (chmette) [inline src/helpers/validateSignUp.js:7]: i love regex :)

Comment (AdrianValdes) [reply src/helpers/validateSignUp.js:7]: I hate ti :)

Comment (AdrianValdes) [reply src/helpers/validateSignUp.js:7]: Obviously I copied that from some where.

File: src/auth/SignUp.jsx [hunk 2]
```diff
@@ -27,10 +28,16 @@ const initialNewUserState = {
 
 export const SignUp = () => {
   const [newUser, setNewUser] = useState(initialNewUserState);
+  const [error, setError] = useState(null);
+
```
Comment (chmette) [inline src/auth/SignUp.jsx:32]: i know here i is the logical part, but we haven' t marked the required fields in the form yet did we?

Comment (AdrianValdes) [reply src/auth/SignUp.jsx:32]: No, no yet, that's the browser on validation we can put it and the end, it makes write in every field.

**Review by chmette — APPROVED** [review]
looks all really good. i noted few questions on my sheet for later

**Pull request merged**